### PR TITLE
fix_OCP-12194 outdated commands

### PR DESCRIPTION
**Pull request by weinliu**
@jhou1 @sunilcio @lyman9966 could you help take a look? 

log: https://privatebin-it-iso.int.open.paas.redhat.com/?2c88b39b06c8c77e#cSjMvYDjUD/viYiJN7wupzzIXFkgfTnX047QGzKB5YU=

### CODE REVIEW

**Comment by weinliu** [comment]
@jhou1 @sunilcio @lyman9966 could you help? Thanks,

File: features/cli/oc_volume.feature [hunk 1]
```diff
@@ -1,24 +1,21 @@
...
   @smoke
   Scenario: Create a pod that consumes the secret in a volume
     Given I have a project
-    When I run the :secrets client command with:
-      | action         | new-basicauth     |
-      | name           | basicsecret       |
-      | username       | user-1            |
-      | password       | pass-1            |
+    Given I run the :create client command with:
+          | f | <%= BushSlicer::HOME %>/testdata/pods/allinone-volume/secret.yaml |
```
Comment (pruan-rht) [inline features/cli/oc_volume.feature:10]: @weinliu indentation issue

File: features/cli/oc_volume.feature [hunk 2]
```diff
@@ -1,24 +1,21 @@
...
     Then the step should succeed
-    When I run the :secrets client command with:
-      | action         | add                    |
-      | serviceaccount | serviceaccount/default |
-      | secrets_name   | secret/basicsecret     |
+    When I run the :secret_link client command with:
+          | secret_name | test-secret |
```
Comment (pruan-rht) [inline features/cli/oc_volume.feature:13]: @weinliu indentation issue

File: features/cli/oc_volume.feature (hunk 1 above)
Comment (weinliu) [reply features/cli/oc_volume.feature]: @pruan-rht fixed. Thanks!

File: features/cli/oc_volume.feature (hunk 2 above)
Comment (weinliu) [reply features/cli/oc_volume.feature]: @pruan-rht fixed. Thanks!

File: features/cli/oc_volume.feature [hunk 3]
```diff
@@ -1,24 +1,21 @@
...
-      | username       | user-1            |
-      | password       | pass-1            |
+    Given I run the :create client command with:
+      | f | <%= BushSlicer::HOME %>/testdata/pods/allinone-volume/secret.yaml |
     Then the step should succeed
-    When I run the :secrets client command with:
-      | action         | add                    |
-      | serviceaccount | serviceaccount/default |
-      | secrets_name   | secret/basicsecret     |
+    When I run the :secret_link client command with:
```
Comment (liangxia) [inline features/cli/oc_volume.feature:12]: Does the change works on old 4.x, e.g, 4.2 ?

Comment (weinliu) [reply features/cli/oc_volume.feature:12]: @liangxia  Yes

```
$ oc secrets link -h
Link secrets to a service account

 Linking a secret enables a service account to automatically use that secret for some forms of authentication.

Usage:
  **oc secrets link serviceaccounts-name secret-name [another-secret-name]... [flags]**

Examples:
  # Add an image pull secret to a service account to automatically use it for pulling pod images:
  oc link serviceaccount-name pull-secret --for=pull

  # Add an image pull secret to a service account to automatically use it for both pulling and pushing build images:
  oc link builder builder-image-secret --for=pull,mount

  # If the cluster's serviceAccountConfig is operating with limitSecretReferences: True, secrets must be added to the
pod's service account whitelist in order to be available to the pod:
  oc link pod-sa pod-secret

Options:
      --for=[mount]: type of secret to link: mount or pull

Use "oc options" for a list of global command-line options (applies to all commands).
[weinliu@vm251-93 verification-tests]$ oc version
Client Version: 4.4.0-rc.10
**Server Version: 4.2.0-0.nightly-2020-06-01-074447**
Kubernetes Version: v1.14.6+32e1868
```

Comment (pruan-rht) [reply features/cli/oc_volume.feature:12]: /lgtm

**Pull request merged**